src/ildasm/dis.cpp

```diff
@@ -1646,7 +1646,7 @@ BOOL Disassemble(IMDInternalImport *pImport, BYTE *ILHeader, void *GUICookie, md
 
             case ShortInlineBrTarget:
             {
-                char offset = (char) pCode[PC];
+                signed char offset = (signed char) pCode[PC];
```

Comment:
`pCode` is `BYTE`, which is `unsigned char` or `uint8`.
`long offset = pCode[PC]` or `long offset = (long) pCode[PC]` will revive the bug.